config/flags_service_start.go

```diff
@@ -69,6 +69,13 @@ var (
 		Usage: "Comma separated list of active services.",
 		Value: strings.Join([]string{"wireguard", "scraping", "data_transfer", "dvpn"}, ","),
 	}
+
+	// FlagStoppedServices a comma-separated list of stopped services.
+	FlagStoppedServices = cli.StringFlag{
+		Name:  "stopped-services",
+		Usage: "Comma separated list of stopped services.",
+		Value: strings.Join([]string{}, ","),
```

Comment:
OK, I'm giving up. Mark this flag at least as hidden. 
```suggestion
		Value: strings.Join([]string{}, ","),
		Hidden: true,
```